LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_2Jets_NewVtx.xml

```diff
@@ -0,0 +1,137 @@
+<marlin>
+
+<execute>
+	<processor name="JetClusteringAndFlavorTag"/>
+</execute>
+
+<global>
+	<parameter name="LCIOInputFiles"> 
+__INPUTFILES__
+        </parameter>
+	<parameter name="MaxRecordNumber" value="0" />  
+	<parameter name="SkipNEvents" value="0" />  
+	<parameter name="SupressCheck" value="false" />  
+	<parameter name="GearXMLFile"> 
+__GEARFILE__
```

Comment:
Why do you have to setup a Gear file ? Gear has been deprecated for the IDR production. You normally don't need it

Reply:
Thank you for spotting this part. I didn't pay attention for that part but I confirmed that indeed the magnetic field has been taken care of throught MarlinUtil. So I remove those parts.